src/google/protobuf/util/json_util.h

```diff
@@ -48,8 +48,11 @@ namespace util {
 struct JsonParseOptions {
   // Whether to ignore unknown JSON fields during parsing
   bool ignore_unknown_fields;
+  // Whether to ignore unrecognized values in enum fields
+  bool ignore_unknown_enum_values;
```

Comment:
Instead of adding a new flag, could you update the behavior of the existing ignore_unknown_fields behvaior to ignore unknown enum values as well?

Reply:
Sure, I'll do

Reply:
I updated my pull request. Now JsonParseOptions::ignore_unknown_files allows ignoring unknown enum values as well. No need for additional option.